src/scrapers/base-isracard-amex.ts

```diff
@@ -187,8 +188,9 @@ function convertTransactions(txns: ScrapedTransaction[], processedDate: string):
       date: txnMoment.toISOString(),
       processedDate: currentProcessedDate,
       originalAmount: isOutbound ? -txn.dealSumOutbound : -txn.dealSum,
-      originalCurrency: convertCurrency(txn.currencyId),
+      originalCurrency: convertCurrency(txn.currentPaymentCurrency),
```

Comment:
Fallback to `currencyId` if `currentPaymentCurrency` is nullish

```suggestion
      originalCurrency: convertCurrency(txn.currentPaymentCurrency ?? txn.currencyId),
```